src/main/java/me/jellysquid/mods/sodium/client/gui/options/control/SliderControl.java

```diff
@@ -157,6 +158,11 @@ public boolean mouseDragged(double mouseX, double mouseY, int button, double del
 
             return false;
         }
+
+        @Override
+        public void reset() {
+            this.thumbPosition = this.getThumbPositionForValue(option.getValue());
```

Comment:
I have just come to realize how needless that interface was, you can simply call whatever you need to reset on the render method. In `SliderControl`'s case I would place it in `renderSlider` method. Same thing applies for `CyclingControl`.

Reply:
I have removed the interface in 4fda9e5ac2e541358b45c48416e3c3f273e530ae.